.github/workflows/close_pr.yaml

```diff
@@ -15,10 +15,10 @@ jobs:
       - uses: humanitec/setup-cli-action@v1
         with:
           version: ${{ env.HUMCTL_VERSION }}
-      
+
       - name: Delete Humanitec Env
         run: |
           humctl delete env pr-${{ github.event.number }} \
               --token ${{ secrets.HUMANITEC_TOKEN }} \
               --org ${{ vars.HUMANITEC_ORG }} \
-              --app ${{ vars.APP_NAME }}
+              --app ${{ vars.HUMANITEC_APP }}
```

Comment:
`HUMANITEC_APP` is the standard name to set your local context for humctl and the "name" is actually something different, hence I think using `HUMANITEC_APP` would be better.